### PR TITLE
Fix: Add keyboard layout support to bastion kickstart template

### PR DESCRIPTION
**Pull request by AmadeusPodvratnik**
signed-off-by: Amadeuds Podvratnik [pod@de.ibm.com](mailto:pod@de.ibm.com)

This fix adds support for the keyboard layout to the kickstart file to the create bastion playbook.

### CODE REVIEW

**Comment by AmadeusPodvratnik** [comment]
@smolin-de Can you pls review this little PR. Thx.

**Comment by smolin-de** [comment]
> @smolin-de Can you pls review this little PR. Thx.

1. The commit message should start with "fix:". 
2. The commit message body looks strange. It tsrats with "Test for create bastion:
TASK [create_bastion : Waiting 1...."
3.

**Review by smolin-de — COMMENTED** [review]
see my comments

File: inventories/default/group_vars/all.yaml.template [hunk 1]
```diff
@@ -184,6 +184,7 @@ env:
 # Section 14 - (Optional) Misc
   language: en_US.UTF-8
   timezone: America/New_York
+  keyboard: us
```
[the remark this answers is not in the document]
Comment (AmadeusPodvratnik) [reply inventories/default/group_vars/all.yaml.template:187]: I would suggest to open a discussion how flexible the bastion kickstart file should be. What kind of parameter we would like to configure comprehensive.

**Review by smolin-de — APPROVED** [review]

**Review by smolin-de — APPROVED** [review]

**Review by smolin-de — APPROVED** [review]

**Review by smolin-de — APPROVED** [review]

**Review by smolin-de — APPROVED** [review]

**Review by smolin-de — APPROVED** [review]

**Review by jacobemery — APPROVED** [review]
lgtm

**Pull request merged**